### PR TITLE
Add 1080p screenshots

### PR DESCRIPTION
**Pull request by SFR-git**
Signed-off-by: Sol Fisher Romanoff <sol@solfisher.com>

CC @vchernin @TheEvilSkeleton: Do these images look acceptable?

### CODE REVIEW

**Review by TheEvilSkeleton — CHANGES_REQUESTED** [review]
So I just looked at the resolution, it looks like there's a 9 pixel difference on both images:

image

Can you fix this?

**Comment by SFR-git** [comment]
That's also the case in the 4K pictures, where 6 pixels are added on each side. I didn't want to cut the image or downscale it improperly. No idea why that happens.

**Comment by TheEvilSkeleton** [comment]
Oh okay, should be good then. Let me check if it works and then I will approve if it's good to go.

**Review by TheEvilSkeleton — APPROVED** [review]

**Comment by panos** [comment]
Since @TheEvilSkeleton the person effectively in charge of the Flatpak package, perhaps it could be a good idea to give them write access to this repository instead of forcing them through a lot of bureaucracy in case some change needs to happen later down the line?

**Comment by SFR-git** [comment]
On Sun Aug 8, 2021 at 3:21 PM IDT, Panagiotis Vasilopoulos wrote:
> Since @TheEvilSkeleton the person effectively in charge of the Flatpak
> package, perhaps it could be a good idea to give them write access to
> this repository instead of forcing them through a lot of bureaucracy in
> case some change needs to happen later down the line?

Not sure about this repo. I think these are one-time additions --
anything else that should change belongs in tenacityteam/tenacity.

**Comment by TheEvilSkeleton** [comment]
I'm okay with multiple people looking over an MR, and I personally prefer it that way. And yeah, they're one-time additions and we won't see a lot of change. I wouldn't mind having write access, though.

**Comment by vchernin** [comment]
> That's also the case in the 4K pictures, where 6 pixels are added on each side. I didn't want to cut the image or downscale it improperly. No idea why that happens.

I just realized the same is true for my original screenshots. I am not sure how that's possible. My monitor is 3840x2160, and I used gnome screenshot's window feature to screenshot. I am not sure how the window can be bigger than the monitor...

If that causes problems I can retake them (or maybe try cropping if it doesn't ruin the screenshot).

I checked what appstream-util (likely used for distro packages) says here (using @TheEvilSkeleton's update metainfo PR).

```
# Without making any changes to the PR (4k screenshots):
$ appstream-util validate tenacity.metainfo.xml 
tenacity.metainfo.xml: FAILED:
• tag-invalid           : Expected children for tag
• attribute-invalid     : <screenshot> width too large [https://cdn.statically.io/gh/tenacityteam/assets/781026d130ebe9b2e48c269d6b355cb576920211/screenshots/tenacity-screenshot-light-3840x2160.png] maximum is 1600px
• attribute-invalid     : <screenshot> height too large [https://cdn.statically.io/gh/tenacityteam/assets/781026d130ebe9b2e48c269d6b355cb576920211/screenshots/tenacity-screenshot-light-3840x2160.png] maximum is 900px
• attribute-invalid     : <screenshot> width too large [https://cdn.statically.io/gh/tenacityteam/assets/f2bf1eb302d3b5af37b64c9b27dfae3123ac885a/screenshots/tenacity-screenshot-dark-3840x2160.png] maximum is 1600px
• attribute-invalid     : <screenshot> height too large [https://cdn.statically.io/gh/tenacityteam/assets/f2bf1eb302d3b5af37b64c9b27dfae3123ac885a/screenshots/tenacity-screenshot-dark-3840x2160.png] maximum is 900px
• tag-missing           : <release> required
Validation of files failed

# Running validate-relax instead (still 4k screenshots):
$ appstream-util validate-relax tenacity.metainfo.xml 
tenacity.metainfo.xml: FAILED:
• tag-invalid           : Expected children for tag
• attribute-invalid     : <screenshot> width too large [https://cdn.statically.io/gh/tenacityteam/assets/781026d130ebe9b2e48c269d6b355cb576920211/screenshots/tenacity-screenshot-light-3840x2160.png] maximum is 3200px
• attribute-invalid     : <screenshot> height too large [https://cdn.statically.io/gh/tenacityteam/assets/781026d130ebe9b2e48c269d6b355cb576920211/screenshots/tenacity-screenshot-light-3840x2160.png] maximum is 1800px
• attribute-invalid     : <screenshot> width too large [https://cdn.statically.io/gh/tenacityteam/assets/f2bf1eb302d3b5af37b64c9b27dfae3123ac885a/screenshots/tenacity-screenshot-dark-3840x2160.png] maximum is 3200px
• attribute-invalid     : <screenshot> height too large [https://cdn.statically.io/gh/tenacityteam/assets/f2bf1eb302d3b5af37b64c9b27dfae3123ac885a/screenshots/tenacity-screenshot-dark-3840x2160.png] maximum is 1800px
Validation of files failed
```

When the 4k screenshots are replaced by 1080p ones:
```
$ appstream-util validate tenacity.metainfo.xml 
tenacity.metainfo.xml: FAILED:
• tag-invalid           : Expected children for tag
• attribute-invalid     : <screenshot> width too large [https://raw.githubusercontent.com/tenacityteam/assets/3450b637edb41c3a84aad6b2047d0c76af4b7639/screenshots/tenacity-screenshot-light-1920x1080.png] maximum is 1600px
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/tenacityteam/assets/3450b637edb41c3a84aad6b2047d0c76af4b7639/screenshots/tenacity-screenshot-light-1920x1080.png] maximum is 900px
• attribute-invalid     : <screenshot> width too large [https://raw.githubusercontent.com/tenacityteam/assets/3450b637edb41c3a84aad6b2047d0c76af4b7639/screenshots/tenacity-screenshot-dark-1920x1080.png] maximum is 1600px
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/tenacityteam/assets/3450b637edb41c3a84aad6b2047d0c76af4b7639/screenshots/tenacity-screenshot-dark-1920x1080.png] maximum is 900px
• tag-missing           : <release> required
Validation of files failed
```

```
⬢[v@toolbox help]$ appstream-util validate-relax tenacity.metainfo.xml 
tenacity.metainfo.xml: FAILED:
• tag-invalid           : Expected children for tag
Validation of files failed
```

So having 1080p screenshots passes `validate-relax` but not `validate` while 4k ones pass neither. I am not sure if the 1080p screenshots would work or not for distro packages. Downscaling to 1600x900 doesn't sound ideal.
[
Thankfully Fedora at least only requires](https://docs.fedoraproject.org/en-US/packaging-guidelines/AppData/) `validate-relax`, so maybe others do the same.

[Debian requires using a different command](https://wiki.debian.org/AppStream/Guidelines) to validate: (why?)
```
# with 1080p screenshots
$ appstreamcli validate tenacity.metainfo.xml 
I: org.tenacityaudio.Tenacity:9: summary-has-dot-suffix
     Audio editor and recorder. Nothing more, nothing less.

Validation was successful: infos: 1, pedantic: 2

# with 4k screenshots
$ appstreamcli validate tenacity.metainfo.xml 
I: org.tenacityaudio.Tenacity:9: summary-has-dot-suffix
     Audio editor and recorder. Nothing more, nothing less.

Validation was successful: infos: 1, pedantic: 2
```

tl;dr if you're confused as I was when I wrote this:
So even though apparently every distro does this differently, it appears using 1080p screenshots in metainfo would satisfy at least Fedora, Flathub and Debian. Fedora would not accept the 4k ones. 

Also this raises another question. Seeing as ideally there is one metainfo file for all Tenacity packages, does this mean we should use 1080p screenshots exclusvely in metainfo, including for Flathub and Debian, even though Flathub and Debian accept 4k ones?

**Comment by TheEvilSkeleton** [comment]
I wouldn't mind 1080p because previews are generally small.

Here's how Flathub looks on Audacity:

image

Here's how GNOME Software looks on Audacity:

image

So even doing 900p won't be a problem because previews are small enough to make a difference.

**Comment by SFR-git** [comment]
I think 1080p's good, but it's worth keeping 4K around just so we have it -- once we fix the mysterious pixels, of course.

**Comment by vchernin** [comment]
So it seems there definitely is a bug with GNOME Screenshot somehow, the window mode seems to always create extra pixels.

Using the whole screen feature works properly (gives 3840x2160), but then GNOME's top bar shows up which probably isn't ideal for Tenacity. (I can't put it on the second monitor since then that would show up too in the screenshot).

Screenshot from 2021-08-08 14-46-52

However, I am able to use the selection tool (and just manually drag from corner to corner), while using my second monitor to place the top bar. This gives as far as I can tell an ideal screenshot.

Screenshot from 2021-08-08 14-50-00

Should I retake the screenshots with this method at 4k, 1080p, and maybe 1600p (if I can) and make a PR for those? I'll make sure to include a nice waveform and use higher scaling for a visible UI as before.

**Comment by TheEvilSkeleton** [comment]
@vchernin you should use Adwaita dark when using Tenacity in dark mode. It should look a lot better IMO.

**Comment by vchernin** [comment]
Hm, I used [HighInverseContrast in my last one](https://github.com/tenacityteam/assets/pull/4). That one looked best to me.

**Comment by TheEvilSkeleton** [comment]
I guess you could use that one then. The white dropdown buttons on a dark theme look really ugly.

**Comment by TheEvilSkeleton** [comment]
Any update on this?
